CMakeLists.txt

```diff
@@ -173,6 +173,7 @@ endif()
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++${META_CXX_STD} -pedantic")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Woverloaded-virtual")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g")
```

Comment:
I prefer debuginfo on all builds to be honest.  I feel like that's worth it, and doesn't come at too much of a cost here (small library, not going to bloat size or compile time too much).